R/FilterStateRange.R

```diff
@@ -128,25 +128,25 @@ RangeFilterState <- R6::R6Class( # nolint
     #' @description
     #' Initialize a `FilterState` object for range selection.
     #' @param x (`numeric`)
-    #'   values of the variable used in filter
+    #'   variable to be filtered
     #' @param x_reactive (`reactive`)
     #'   returning vector of the same type as `x`. Is used to update
     #'   counts following the change in values of the filtered dataset.
     #'   If it is set to `reactive(NULL)` then counts based on filtered
     #'   dataset are not shown.
     #' @param slice (`teal_slice`)
-    #'   object created using [teal_slice()]. `teal_slice` is stored
-    #'   in the class and `set_state` directly manipulates values within `teal_slice`. `get_state`
-    #'   returns `teal_slice` object which can be reused in other places. Beware, that `teal_slice`
-    #'   is a `reactiveValues` which means that changes in particular object are automatically
-    #'   reflected in all places which refer to the same `teal_slice`.
+    #'   specification of this filter state.
+    #'   `teal_slice` is stored in the object and `set_state` directly manipulates values within `teal_slice`.
+    #'   `get_state` returns `teal_slice` object which can be reused in other places.
+    #'   Note that `teal_slice` is a `reactiveValues`, which means it has reference semantics, i.e.
+    #'   changes made to an object are automatically reflected in all places that refer to the same `teal_slice`.
     #' @param extract_type (`character`)
-    #' whether condition calls should be prefixed by `dataname`. Possible values:
+    #'   specifying whether condition calls should be prefixed by `dataname`. Possible values:
     #' - `character(0)` (default) `varname` in the condition call will not be prefixed
     #' - `"list"` `varname` in the condition call will be returned as `<dataname>$<varname>`
     #' - `"matrix"` `varname` in the condition call will be returned as `<dataname>[, <varname>]`
     #'
-    #' @param ... additional arguments to be saved as a list in `private$extras` field
+    #' @return Returns `self`, invisibly.
```

Comment:
Same comment as above

```suggestion
    #' @return `self`, invisibly.
```